demisto_sdk/commands/content_graph/objects/integration.py

```diff
@@ -96,6 +96,13 @@ def to_raw_dict(self) -> Dict:
         remove_nulls_from_dictionary(command)
         return command
 
+    @property
+    def outputs_context_paths(self) -> list:
+        """
+        This method returns a list of the all command's outputs context path.
+        """
+        return [output.contextPath for output in self.outputs]
+
```

Comment:
```suggestion
```
As we spoke, let's check by command. For each command, check whether its old outputs are contained in the new ones.